docs/docs/design/availability-distribution.md

```diff
@@ -0,0 +1,174 @@
+# Availability Distribution Subsystem
+
+[The availability distribution subsystem](https://paritytech.github.io/polkadot-sdk/book/node/availability/availability-distribution.html)
+is responsible for retrieving and distributing availability data such as PoVs and erasure chunks.
+
+## Subsystem Structure
+
+The implementation must conform to the `Subsystem` interface defined in the `parachaintypes` package. It should live in
+a package named `availabilitydistribution` under `dot/parachain/availability-distribution`.
```

Comment:
how about naming it avdistr? by example of strconv or syscall

Reply:
Fine with me. So far we've always used the full name for the package of parachain subsystems.

Reply:
Then I guess we should follow what we already have, since it is a discussion we cant hold as a part of this review.